the package needs to be updated to remove the .git directory from the package

**error:**
`
npm ERR! code EISGIT
npm ERR! path /Users/guilhermeramos/Desktop/projects/fillgi-react-native/node_modules/react-native-login-twitter
npm ERR! git /Users/guilhermeramos/Desktop/projects/fillgi-react-native/node_modules/react-native-login-twitter: Appears to be a git repo or submodule.
npm ERR! git     /Users/guilhermeramos/Desktop/projects/fillgi-react-native/node_modules/react-native-login-twitter
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/guilhermeramos/.npm/_logs/2020-03-18T03_23_28_740Z-debug.log
`

ref to fix it:
https://github.com/APSL/react-native-version-number/issues/13

--- a/.npmignore
+++ b/.npmignore
@@ -27,6 +27,7 @@ DerivedData
 # Node
 node_modules
 
+.git
 
 ### Gradle ###
 .gradle

